### ISSUE TITLE
The extension breaks the autogenerated MediaWiki API help page

### ISSUE THREAD

**Issue by ankostis** [gesinn-it/QRLite#3]
The error i get when visiting `http://vela.jrc.it/w/api.php?action=help&modules=main` is this:
```
Internal Server Error
[94569103dfd0326b2a83d65e] /w/api.php?action=help&modules=main Error from line 200 of /var/lib/w/includes/api/ApiModuleManager.php: Class 'QRLiteApi' not found
Backtrace:
#0 /var/lib/w/includes/api/ApiModuleManager.php(167): ApiModuleManager->instantiateModule(string, string, NULL)
#1 /var/lib/w/includes/api/ApiBase.php(623): ApiModuleManager->getModule(string)
#2 /var/lib/w/includes/api/ApiBase.php(2328): ApiBase->getModuleFromPath(string)
#3 /var/lib/w/includes/api/ApiHelp.php(429): ApiBase->getFinalParamDescription()
#4 /var/lib/w/includes/api/ApiHelp.php(146): ApiHelp::getHelpInternal(DerivativeContext, array, array, array)
#5 /var/lib/w/includes/api/ApiHelp.php(50): ApiHelp::getHelp(DerivativeContext, array, array)
#6 /var/lib/w/includes/api/ApiMain.php(1579): ApiHelp->execute()
#7 /var/lib/w/includes/api/ApiMain.php(535): ApiMain->executeAction()
#8 /var/lib/w/includes/api/ApiMain.php(506): ApiMain->executeActionWithErrorHandling()
#9 /var/lib/w/api.php(94): ApiMain->execute()
#10 {main}
```

It seems the `QRLiteApi` class is missing, but still declared in  https://github.com/gesinn-it/QRLite/blob/f83c77aa92f7a3f291eaff896e5906ff84a5fa46/extension.json#L12-L15

If i delete these 3 lines from `extension.json` the problem is fixed.

**Comment by gesinn-it** [comment]
Thanks for reporting this issue. There are some more things that need some clean-up. Stay tuned...

**Comment by ankostis** [comment]
Thank you for this project.

Is there some issue for Request for Enhancements?  Like providing a wau to open a customizeable "print page" ?

**Comment by gesinn-it** [comment]
Feel free to open new issues for your feature wishes or create pull requests.

**Comment by gesinn-it** [comment]
closed by release 0.3.2
